src/Groups/matrices/matrix_manipulation.jl

```diff
@@ -170,14 +170,6 @@ end
 #
 ########################################################################
 
-
-Base.:*(v::AbstractAlgebra.Generic.FreeModuleElem{T},x::MatElem{T}) where T <: RingElem = v.parent(v.v*x)
```

Comment:
Unfortunately we use this in one book example in the group theory chapter:
```
julia> R2 = free_module(K, 2) # the "euclidean" plane over K
Vector space of dimension 2 over QQBar

julia> A = R2([0,1])
(Root 0 of x, Root 1.00000 of x - 1)

julia> pts = [A*mat_rot^i for i in 0:4];
```

However if we replace `A*mat_rot ` by `A*G_mat[1]` it is fine because we retained that method -- we can retain it as (a) it is not type piracy, and (b) matrix group elements are guaranteed to be square).

So we either need to modify the book example or keep this method... Guess we can decide it when we decide how to proceed with the book.

Reply:
Since the book is now being revised I can adjust the book example to match, I guess. But I suggest that is done first in the book, and then in all repositories which have stuff derived from it. And only once that all is through do we merge this here.

Repositories that contain code extract from book examples
- Oscar.jl of course
- https://github.com/oscar-system/book-notebooks (do we still care about it?)
- https://github.com/oscar-system/book.oscar-system.org
- anything else?

Reply:
This really needs to be settled one way or another this Wednesday.

Reply:
I discussed this with @fieker this morning and it turns out we don't even need the linke `pts = [A*mat_rot^i for i in 0:4];` in the book example anymore! So I have now taken it out of the book, and that means this obstacle is gone.

Just need to wait for the book examples to be synced into this repository